data/src/main/java/com/microsoft/azure/kusto/data/ClientRequestProperties.java

```diff
@@ -301,17 +319,28 @@ public Map<String, String> getTracingAttributes() {
         return attributes;
     }
 
+    /**
+     * Gets the amount of time a query may execute on the service before it times out, formatted as a KQL timespan.
+     * @param timeoutObj amount of time before timeout, which may be a Long, String or Integer.
+     *                    Value must be between 1 minute and 1 hour, and so value below the minimum or above the maximum will be adjusted accordingly.
+     */
+    // TODO: Breaking change for next major release: static String getTimeoutAsTimespan(Object timeoutObj)
```

Comment:
Note that this method returns a `timespan`, not just any `String`.

Also, similar existing methods already use this kind of naming.

Reply:
I'll add the replacement, too: `String getTimeoutAsTimespan(Object timeoutObj)`